app/views/ListView.scala.html

```diff
@@ -91,9 +91,11 @@ <h2 class="govuk-heading-m">@renderMessage(heading)</h2>
                 }
             }
 
-            @viewmodel.page.paginatedViewModel.map { paginatedViewModel =>
-                @govukPagination(PaginationViewModel(paginatedViewModel.pagination))
-            }
+                   @viewmodel.page.paginatedViewModel.map { paginatedViewModel =>
```

Comment:
Indentation looks wrong, but that can be addressed in the next PR to spare some build time.